doc/terminology.md

```diff
@@ -0,0 +1,83 @@
+# Terminology
+
+This pages gives an overview about the terms we use, when talking about KUKSA.val components, architectures and the data handled by KUKSA.val.
+
+## Basic elements in a KUKSA.val deployment
+
+![Basic architecture](./pictures/terminology.svg)
+
+KUKSA.val helps you providing and interacting with VSS data in-vehicle. At the center 
+
+## VSS Ressource Server
+A server that provides the KUSKA.val APIs to interact with VSS signals.
+
+Usually, for KUKSA.val this is the KUKSA.val databroker. The server keeps the current state of all known VSS signals and enables clients to get information regarding those singals or changing their state. It is also responsible of for checking whether a client has sufficient authorisation to access certain signals.
+
+Architecture-wise most things in this document apply to the older KUKSA.val val-server as well. (see [here](./server-vs-broker.md) for technical differences between these two VSS servers).
+
+Databroker's API is GRPC-based, which means it can be accessed over the network by other programs or systems.
+
+
+## Client
+Any application that connects to the server and uses the KUKSA.VAL APIs is a client.
+
+From an architecture point of view, there are  different types of clients, which will be discussed in the following. 
+
+
+For specific examples of components fullfilling these roles also check [System Architecture](./system-architecture.md).
+**todo: probably system architecture not a good name for that file, and I did not update it with the terms in this doc**
+
+## Clients: VSS Consumers
+ VSS Consumers can also be seen as northbound clients of the VSS ressource server. A VSS consumer intends to interact with the Vehicle represented by the VSS model served by the server: A consumer typically wants to read sensors representing vehicle state available in VSS or affecting changes in the vehicle hardware by interacting with actuators available in VSS.
+
+ So typically A VSS consumer might be "apps" running on vehicle computers, the infotainment, or maybe even a user's personal device. 
+
+
+ ## Clients: VSS Providers
```

Comment:
Is this supposed to be a heading - if so remove first blank

Reply:
Done